admin/assembly-admin-templates.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-admin-templates"]
+= Managing templates
+:context: assembly-admin-templates
+
+toc::[]
+
+As an administrator, you can create and add templates for a {product} instance.
+Templates enable developers to easily create components, and then publish these components to different locations, such as GitHub or GitLab.
```

Comment:
I think that we should clarify what a `component` is. WDYT ?

As I suspect that this sentence is coming from the backstage documentation, we should be careful when reusing the upstream documentation as content has been designed long time ago, is probably outdated, etc.

From my point of view, this is how we should introduce `Templates`

As a developer, you can create `Template` able to generate from skeleton of files using inputs, resources processed by a templating engine - nunjucks : https://github.com/mozilla/nunjucks and to publish them within the product software catalog, git repository (github, gitlab, etc).

A template is a form composed of different ui fields (input, checkboxes, textarea, radio buttons, etc) that you describe in a YAML file (add link to the doc: https://backstage.io/docs/features/software-templates/writing-templates) and include actions which are, like in a github workflow, steps executed in a sequential order. The steps can be executed conditionally

The list of the actions installed within the product can be viewed using the following link `https://<RHDP_INSTANCE>/create/actions`. 

For development purposes, you can use the `Template editor` using the address `https://<RHDP_INSTANCE/create/edit` to create a new template, import your template or to consult the list of the existing ui fields.

**Note**: New actions can be developed - add link to the doc: https://backstage.io/docs/features/software-templates/writing-custom-actions like custom fields - https://backstage.io/docs/features/software-templates/writing-custom-field-extensions

Reply:
>As I suspect that this sentence is coming from the backstage documentation, we should be careful when reusing the upstream documentation as content has been designed long time ago, is probably outdated, etc.

Is there a reason why we as a team are not making an effort to update docs in the upstream project first, if we know that they are outdated? At RH it's usually considered important to work [upstream first](https://www.redhat.com/en/blog/what-open-source-upstream) and I feel like the Backstage docs are probably a good opportunity to do this.